### PR TITLE
Update table headers that display for sal3 batches summary

### LINKED ISSUE #452: Update columns that display for sal3 batches summary
Date Needed
Name or short description
Stop code
\# of Items Requested
Status
Indication as to whether it is a 1 time request or multiple requests
Date of actual request

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #452

### CODE REVIEW

**Comment by jgreben** [comment]
@dlrueda this is ready to be merged.

File: app/views/sal3_batch_requests_batches/index.html.erb [hunk 1]
```diff
@@ -2,23 +2,29 @@
...
-        <td class='col-xs-1'><%= link_to batch.batch_id, batch %></td>
-        <td class='col-xs-2'><%= batch.batch_name %></td>
-        <td class='col-xs-3'><%= batch.user_name %></td>
-        <td class='col-xs-4'><%= batch.user_email %></td>
-        <td class='col-xs-5'><%= batch.comments %></td>
-        <td class='col-xs-6'><%= link_to 'Edit', edit_sal3_batch_requests_batch_path(batch), class: 'btn btn-primary' %></td>
-        <td class='col-xs-6'><%= link_to 'Show', sal3_batch_requests_batch_path(batch), class: 'btn btn-primary' %></td>
+        <td class='col-xs-1'><%= link_to 'Batch details', sal3_batch_requests_batch_path(batch), class: 'btn btn-default' %></td>
+        <td class='col-xs-2'><%= batch.batch_needbydate.strftime('%x') %></td>
+        <td class='col-xs-3'><%= batch.comments %></td>
```
Comment (dlrueda) [inline app/views/sal3_batch_requests_batches/index.html.erb]: I think this should be batch.batch_name.  There is a separate column in the table for batch.comments, but that's not this.

File: app/views/sal3_batch_requests_batches/index.html.erb [hunk 2]
```diff
@@ -2,23 +2,29 @@
...
+        <td class='col-xs-4'><%= batch.stopcode %></td>
+        <td class='col-xs-5'><%= batch.num_bcs %></td>
+        <td class='col-xs-6'><%= batch.status %></td>
+        <td class='col-xs-7'><%= batch.num_bcs > batch.batch_numpullperday %></td>
+        <td class='col-xs-8'><%= batch.load_date %></td>
+        <td class='col-xs-9'><%= link_to 'Mange batch request', edit_sal3_batch_requests_batch_path(batch), class: 'btn btn-primary' %></td>
```
Comment (dlrueda) [inline app/views/sal3_batch_requests_batches/index.html.erb]: typo: "Mange" should be "Manage"

**Comment by dlrueda** [comment]
Made a couple comments of changes needed...

**Pull request merged**